src/hrflow_connectors/connectors/bullhorn/docs/push_profile.md

```diff
@@ -0,0 +1,60 @@
+# Push profile
+`Hrflow.ai` :arrow_right: `Bullhorn`
+
+`PushProfileAction` pushes a Hrflow.ai profile to `Bullhorn` via their API.
+
+## Parameters
+
+| Field | Type | Description |
+| ----- | ---- | ----------- |
+| `logics`  | `List[str]` | Function names to apply as filter before pushing the data. Default value : `[]`        |
+| `local_scope`  | `Optional[Dict[str, Any]]` | A dictionary containing the current scope's local variables. Default value : `None`        |
+| `global_scope`  | `Optional[Dict[str, Any]]` | A dictionary containing the current scope's global variables. Default value : `None`       |
+| `format_function_name`  | `Optional[str]` | Function name to format job before pushing. Default value : `None`        |
+| `hrflow_client` :red_circle: | `hrflow.Hrflow` | Hrflow client instance used to communicate with the Hrflow.ai API        |
+| `profile` :red_circle: | `Profile` | Profile to push        |
+| `auth` :red_circle: | `OAuth2Session` | Auth instance to identify and communicate with the platform        |
+| `subdomain` :red_circle: | `str` | Subdomain Bullhorn just before bullhornstaffing.com. For example subdomain=my_subdomain.my in http://my_subdomain.my.bullhornstaffing.com/ABC        |
+
+:red_circle: : *required* 
+
+## Example
+Let's take as an example in a [***CATCH workflow***](https://developers.hrflow.ai/docs/workflows#catch-setup).
+```python
+from hrflow_connectors import Bullhorn
+
+from hrflow import Hrflow
+from hrflow_connectors.core.auth import OAuth2Session
+from hrflow_connectors.utils.hrflow import EventParser
+from hrflow_connectors.utils.logger import get_logger_with_basic_config
+
+def workflow(_request, settings):
```

Comment:
We no longer use workflow examples in readmes, use a generic example.
See other connectors readmes for inspiration